yarn.lock

```diff
@@ -13865,6 +14006,37 @@ mocha@^5.1.1:
     mkdirp "0.5.1"
     supports-color "5.4.0"
 
+mockttp@^0.13.0:
```

Comment:
Looks like you need to run `yarn install` to update the lockfile.

Reply:
`yarn install` warns that 
`
info Merge conflict detected in yarn.lock and successfully merged.
warning package-lock.json found. Your project contains lock files generated by tools other than
stencies caused by unsynchronized lock files. To clear this warning, remove package-lock.json.
`
Which program created  `package-lock.json`?  Should I remove it first?

Reply:
done